test/e2e/commands/network.test.mjs

```diff
@@ -103,15 +103,10 @@ describe('NetworkCommand', () => {
         flags.applicationProperties.constName,
         flags.bootstrapProperties.constName,
         flags.chainId.constName,
-        flags.deployHederaExplorer.constName,
```

Comment:
I think we can just remove flags.deployHederaExplorer, since it is now redundant.